packages/combobox/src/Combobox.ts

```diff
@@ -120,6 +120,11 @@ export class Combobox extends Textfield {
 
     private tooltipEl?: Tooltip;
 
+    private resizeObserver: ResizeObserver | undefined;
```

Comment:
This is quite verbose, can you make this optional?
```suggestion
    private resizeObserver?: ResizeObserver | undefined;
```